File: SoloLearn/Tax Free/TaxFree.swift
import Foundation

if let prices = readLine() {
    let array = prices.components(separatedBy: ",")
    var total = 0.0
    for price in array {
        if let p = Double(price) {
            total += p
            if p < 20 {
                total += 0.07 * p
            }
        }
    }
    print(Double(round(total * 100) / 100))
} 

File: SoloLearn/Tax Free/TaxFree.cpp
#include <iostream>
#include <math>
using namespace std;

double price(string value) {
    double price = stod(value);
    if (price < 20)
        price *= 1.07;
    return price
}

int main() {
    string prices;
    double total = 0.0;
    cin >> prices;
    size_t pos = 0;
    while ((pos = prices.find(",")) != string::npos) {
        total += price(prices.substr(0, pos));
        prices.erase(0, pos + 1);
    }
    total += price(prices);
    cout << round(total * 100) / 100;
    return 0;
} 